examples/CMakeLists.txt

```diff
@@ -2,7 +2,15 @@ project(hipsycl-examples)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/../cmake)
 
-find_package(hipSYCL CONFIG REQUIRED)
+find_package(hipSYCL CONFIG)
+
+if(NOT hipSYCL_FOUND)
+  message(WARNING "Not found hipSYCL cmake integration retrying with default path: /opt/hipSYCL/lib/cmake/")
+  set(hipSYCL_DIR /opt/hipSYCL/lib/cmake/)
+  find_package(hipSYCL CONFIG REQUIRED)
```

Comment:
I think we can omit that step if we give cmake a hint to where the package is found in the first `find_package` using the `PATHS` argument. See https://cmake.org/cmake/help/latest/command/find_package.html

Reply:
Ohh I see. I should have read the manual thoroughly beforehand. I think using the PATHS hint makes this change very straightforward. In my opinion it elminates all the other problems. I pushed a new version using the PATHS hint.